### PR TITLE
chore(empty-react-project

### PR DESCRIPTION
**Pull request by mwisemarierose**
## What does this PR do?

Creates  react project from scratch using Webpack 4, Babel, and the Airbnb linting option

## How should this be manually tested?
clone this repository
npm i
npm run start

## What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2569105

### CODE REVIEW

**Review by Placide123 — APPROVED** [review]

**Review by Placide123 — CHANGES_REQUESTED** [review]
other is working fine but if app.js is removed in the component folder it can be good and put it into src

**Review by uwambajeddy — APPROVED** [review]

**Review by joshwambere — APPROVED** [review]
assets folder contains static assets like images videos icons etc, may be you can remove index.js and add random image or icon

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class App extends Component {
```
Comment (joshwambere) [inline src/components/App.js]: use functional components instead of class based components

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (joshwambere) [inline .eslintrc.json:1]: you have lint in the project and yet you pushed codes which has linting errors, and linting was not working as expected

**Review by joshwambere — CHANGES_REQUESTED** [review]

**Review by joshwambere — APPROVED** [review]

**Review by Placide123 — APPROVED** [review]

**Pull request merged**